modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/public-footer/index.html

```diff
@@ -1,125 +1,125 @@
-<footer class="bg-brand-primary-darken-5 footer public-footer">
+<div class="bg-brand-primary-darken-5 c-py-10 footer public-footer">
```

Comment:
Regarding the semantical HTML, this should continue as **footer**.

Thinking about all Liferay users, we need to use footer tag here

![image](https://user-images.githubusercontent.com/22279592/145729542-64d1602b-872a-4da9-bbfd-137ce9b53839.png)